src/collection.ts

```diff
@@ -346,6 +215,175 @@ export class Collection<T extends TigrisCollectionType> {
 		return;
 	}
 
+	/**
+	 * Consume events from a "topic"
+	 *
+	 * @param callback - Callback to consume events asynchronously
+	 */
+	events(callback: EventsCallback<T>) {
```

Comment:
Do we want to support CDC changelog on Topics? If yes, then I'll move this under ReadOnlyCollection

Reply:
No we don't